packages/expo-cli/src/credentials/local/update.ts

```
@@ -0,0 +1,157 @@
+import fs from 'fs-extra';
+import path from 'path';
+import prompts from 'prompts';
+
+import log from '../../log';
+import { Context } from '../context';
+
+type Platform = 'android' | 'ios' | 'all';
+
+export async function updateLocalCredentialsJsonAsync(projectDir: string, platform: Platform) {
+  const ctx = new Context();
+  await ctx.init(projectDir);
+  if (!ctx.hasProjectContext) {
+    throw new Error('project context is required'); // should be checked earlier
+  }
+  if (['all', 'android'].includes(platform)) {
+    log('Updating Android credentials in credentials.json');
+    await updateAndroidAsync(ctx);
+  }
+  if (['all', 'ios'].includes(platform)) {
+    log('Updating iOS credentials in credentials.json');
+    await updateIosAsync(ctx);
+  }
+}
+
+async function updateAndroidAsync(ctx: Context) {
+  const credentialsJsonFilePath = path.join(ctx.projectDir, 'credentials.json');
+  let rawCredentialsJsonObject: any = {};
+  if (await fs.pathExists(credentialsJsonFilePath)) {
+    try {
+      const rawFile = await fs.readFile(credentialsJsonFilePath);
+      rawCredentialsJsonObject = JSON.parse(rawFile.toString());
+    } catch (error) {
+      log.error(`There was an error while reading credentials.json [${error}]`);
+      log.error('Make sure that file is correct (or remove it) and rerun this command.');
+      throw error;
+    }
+  }
+  const experienceName = `@${ctx.manifest.owner || ctx.user.username}/${ctx.manifest.slug}`;
+  const keystore = await ctx.android.fetchKeystore(experienceName);
+  if (!keystore) {
+    throw new Error('There are no credentials configured for this project on Expo servers');
```

Hmm, I'm not convinced this should throw an error.